apps/voting/app/src/components/VoteActions.js

```diff
@@ -239,6 +242,23 @@ const TokenReference = ({
   </Info>
 )
 
+function BlockNumber({ blockNumber }) {
+  const network = useNetwork()
+
+  return network ? (
+    <Link
+      href={blockExplorerUrl('block', blockNumber, {
+        networkType: network.type,
+      })}
+      title={blockNumber}
```

Comment:
We could remove this since the number is not truncated, or maybe replace it by something like this?

```jsx
title={`See block ${blockNumber} on Etherscan`}
```

Reply:
i removed it :)